docs: entirely translate "Contribute to OSRD" (#78)
DGEXSolutions/osrd-website: content/docs/guides/contribute/_index.en.md

- [**Create an issue**](https://github.com/DGEXSolutions/osrd/issues/new/choose) to discuss your contribution project
- Join our matrix instant messaging room ! **(very soon, it's not yet there)**

### Inquire

> As for any project, each change is based on what has been done before.
> Before making any change, learn about what's already there :

Reviewer: ```suggestion
Just like with any project, changes rely on past work.
Before making changes, it is best to learn about what's already there:
```

> - You can [read the technical documentation]({{< ref "/docs" >}})
> - It's best to read the existing code related to your project before proposing something new
> - You can contact the last developers who have worked on code areas that are of interest to you

Reviewer: ```suggestion
- [read technical documentation]({{< ref "/docs" >}})
- read the existing source code related to your project
- chat with developers who last worked on areas you are interested in
```
